examples/monitor-with-thanos/README.md

```diff
@@ -0,0 +1,43 @@
+# TiDBMonitor with Thanos
+
+This document is to show how to integrate TidbMonitor with [Thanos](https://thanos.io/design.md/).
+
+
+## Install
+
+The following commands is assumed to be executed in this directory.
+
+Install the monitor with thanos sidecar:
+
+```bash
+> kubectl -n <namespace> apply -f tidb-monitor.yaml
+```
+
+Wait for monitor Pods ready:
+
+```bash
+watch kubectl -n <namespace> get pod
+```
+
+If you need to store historical data, you can configure the `objectStorageConfig` field and create the corresponding secret.
```

Comment:
Currently, the monitor yaml already has `objectStorageConfig` configured, should we add a step to create the secret before creating the monitor? And add a description that users do not have to create the secret and configure this field if no need to store the historical data.